setup.py

```diff
@@ -1,43 +1,42 @@
-''' Setup for g2p
-'''
+""" Setup for g2p
+"""
 from os import path
 import datetime as dt
 from setuptools import setup, find_packages
 
-build_no = dt.datetime.today().strftime('%Y%m%d')
+build_no = dt.datetime.today().strftime("%Y%m%d")
 
 # Ugly hack to read the current version number without importing g2p:
 # (works by )
 with open("g2p/_version.py", "r", encoding="utf8") as version_file:
     namespace = {}
     exec(version_file.read(), namespace)
-    VERSION = namespace['VERSION'] + "." + build_no
+    VERSION = namespace["VERSION"] + "." + build_no
 
 this_directory = path.abspath(path.dirname(__file__))
 
-with open(path.join(this_directory, 'README.md'), encoding='utf8') as f:
+with open(path.join(this_directory, "README.md"), encoding="utf8") as f:
     long_description = f.read()
 
-with open(path.join(this_directory, 'requirements.txt'), encoding='utf8') as f:
+with open(
+    path.join(this_directory, "requirements", "requirements.txt"), encoding="utf8"
+) as f:
     REQS = f.read().splitlines()
 
 setup(
-    name='g2p',
-    python_requires='>=3.6',
+    name="g2p",
+    python_requires=">=3.6",
```

Comment:
Is >= 3.6 still true? I thought we decided >=3.7? Or was that only for Studio?

Reply:
g2p still works for 3.6!

Reply:
Ah, OK. Then I guess we should keep it at 3.6 for sure. I suppose there's not motivation to move on unless we want to use 3.7 features.